### ISSUE TITLE
XML scrubbing does not work if no other scrubbing options are set

### ISSUE THREAD

**Issue by scottkleinman** [WheatonCS/Lexos#367]
Users will most likely want to operate on xml first before applying other scrubbing options (especially since those options will garble the xml). But, if you de-select all the default scrubbing option, the xml handling is not applied. I tried setting MUFI conversion with no other options, and the MUFI conversion worked but the XML handling was not applied.

**Comment by evanlaferriere** [comment]
what exactly did you change on scrub.html? It's no longer detecting tags

**Comment by scottkleinman** [comment]
It is on my end. I commented out the "Remove Tags" option (lines 137-140). You could try re-enabling those lines. Also, try resetting your session?

**Comment by evanlaferriere** [comment]
How's this going? Is there still a bug? Could you pull?

**Comment by scottkleinman** [comment]
I just pulled, and now I cannot access the XML handling modal. I see only the Remove Tags check box, and checking it doesn't trigger anything.

Tested in Firefox and Chrome.

**Comment by scottkleinman** [comment]
OK, that was due to some code that was not yet ready to be pushed. I can access the modal now and will begin testing.

**Comment by scottkleinman** [comment]
I tested the following code:

```xml
<name type="person" ref="#bioGeorge0001">Geo&rrot;ge</name>
<teiHeader></teiHeader>
<hi rend="touched-red">Some text</hi>
<pb facs="bau0001.xml"/>
```

I used these options:

1. `name`: Remove Element and All Its Contents
2. `teiHeader`: Remove Element and All Its Contents
3. `hi`: Replace Element's Contents with Attribute Value
4. `pb`: Replace Element's Contents with Attribute Value

The result was 
```xml
<name type="person" ref="#bioGeorge0001">Geo&rrot;ge</name>
<teiHeader></teiHeader>
rend
<pb facs="bau0001.xml"/>
```
So it seems like the algorithm is only matching criterion 3, and there it is replacing the content with the attribute's name, rather than its property.

I'll push my test file `george.xml` to `TestSuite/xmlfiles` in a minute.
